editor/i18n/en/localization.js

```diff
@@ -245,7 +244,7 @@ module.exports = {
         pageTurningSpeed: 'The time required to turn over a page. unit: second',
         pageEvents: 'PageView events callback',
         autoPageTurningThreshold:
-            'Auto page turning velocity threshold. When users swipe the PageView quickly, it will calculate a velocity based on the scroll distance and time, if the calculated velocity is larger than the threshold, then it will trigger page turning',
+            'Auto page turning velocity threshold\nWhen users swipe the PageView quickly, it will calculate a velocity based on the scroll distance and time\nif the calculated velocity is larger than the threshold, then it will trigger page turning',
```

Comment:
The turning velocity threshold for page turning
When user swipes the PageView quickly, a velocity is calculated based on the swipe distance and time.
If the velocity is larger than the threshold, the page will be flipped